src/editor/index.php

```diff
@@ -118,7 +132,8 @@ function simplifyPath( $path ) {
 }
 
 // .. as a fully qualified filename
-$strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+// $strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+$strSchemaFQFilename = realpath($strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename );
```

Comment:
See the thread http://knx-user-forum.de/forum/supportforen/cometvisu/41563-visu_config-xsd-existiert-nicht 

The problem happens when /etc/cometvisu gets symlinked as .../cometvisu/config and the XSD has it's normal place outside .../cometvisu/config. Now the link in the XML refers to it by ".." which is fine when you call the string manipulation of the URL and then switch to the file system. But when you switch to the file system fist and then apply the .. there you'll end in a different directory.